Dependency not working with React v18

# 🐛 bug report
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8 || ^17.0" from react-semantic-ui-datepickers@2.17.0
npm ERR! node_modules/react-semantic-ui-datepickers
npm ERR!   react-semantic-ui-datepickers@"^2.17.0" from the root project

## Description of the problem
After updating react to the latest version (v18), dependency errors appear when trying to start the project.

## How has this issue affected you? What are you trying to accomplish?
Attempting to use this module with react and react-dom v18

I was wondering if simply updating the peer dependencies would solve this issue?

--- a/package.json
+++ b/package.json
@@ -80,7 +80,7 @@
     "typescript": "4.6.3"
   },
   "peerDependencies": {
-    "react": "^16.8 || ^17.0",
+    "react": "^16.8 || ^17.0 || ^18.0",
     "semantic-ui-react": ">=0.75.0"
   },
   "resolutions": {

